common/FileSystem.cpp

```diff
@@ -1732,8 +1732,26 @@ bool FileSystem::CreateDirectoryPath(const char* Path, bool Recursive, Error* er
 		std::wstring tempPath;
 		tempPath.reserve(pathLength);
 
+		// for absolute paths, we need to skip over the path root
+		size_t rootLength = 0;
+		if (Path::IsAbsolute(Path))
+		{
+			const wchar_t* root_start = wpath.c_str();
+			wchar_t* root_end;
+			const HRESULT hr = PathCchSkipRoot(const_cast<wchar_t*>(root_start), &root_end);
+			if (FAILED(hr))
+			{
+				Console.ErrorFmt("PathCchSkipRoot() returned {:08X}", static_cast<unsigned>(hr));
```

Comment:
Should this use Error::SetHResult instead of logging to console?

Reply:
Yes it should, good catch